cli/src/macro_opts.rs

```diff
@@ -0,0 +1,122 @@
+#[macro_export]
+macro_rules! shared_opts_wrapper {
+    // Config dir option
+    (
+        #[with = config_dir $(,$modifier:ident)*]
```

Comment:
I find the `#[with = a, b]` a bit unclear given it reads like `with = a` + `b` instead of `with = (a, b)`

would it be possible to replace the coma by a plus sign: `#[with = a + b]` ?

But don't bother too much if it's not a trivial change (or if you don't find it more readable), as this is a minor readability improvement ^^

Reply:
Currently, that not possible rust-lang/rust#18700